styles/widgets/material/actionButton.material.less

```diff
@@ -0,0 +1,65 @@
+.dx-size-default() {
```

Comment:
We do not need this block if the FAB has the same styles in default and compact size schemes.